README.md

```diff
@@ -15,6 +15,114 @@ bin/cake plugin load Robotusers/Tactician
 
 ## Using the plugin
 
+### CakePHP integration
+
+This plugin provides Controller and Model integration through [Commander](https://github.com/robotusers/commander) library.
+
+Commander is a command bus abstraction library for PHP which enables you to decouple your code from a concrete command bus implementation.
+
+#### Using the Commander (PHP 7.1+)
+
+Install `robotusers/commander`:
+
+```
+composer require robotusers/commander
+```
+
+Set up your controllers:
+
+```php
+use Cake\Controller\Controller;
+use Robotusers\Commander\CommandBusAwareInterface;
+use Robotusers\Commander\CommandBusAwareTrait;
+
+class OrdersController extends Controller implements CommandBusAwareInterface
+{
+   use CommandBusAwareTrait;
+
+   public function makeOrder()
+   {
+        // ...
+        $command = new MakeOrderCommand($data);
+        $this->handleCommand($command);
+        // ...
+   }
+
+}
+```
+
+For more information, read [the docs](https://github.com/robotusers/commander/blob/master/README.md).
+
+#### Application hook (CakePHP 3.3+)
+
+If your application supports middleware you can configure the command bus using an application hook.
```

Comment:
What does this mean? That things are available everywhere via "handleCommand"?

Reply:
This means that the command bus configured in a hook will be injected into controllers and models that implement the `CommandBusAwareInterface`.

Reply:
Here's a clarification on that https://github.com/robotusers/cakephp-tactician/commit/574b3cea457db29f578aefebaa88dc909ec4f710